src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
+  border: 1px solid $greyElements;
+  background: $whiteElements;
+  box-sizing: border-box;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    width: 160px;
+    height: 134px;
+    background-size: 100% 100%;
+    margin: 32px 19px 40px;
+  }
+
+  &__name {
+    @include marginSetings;
+    width: 166px;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainColor;
+  }
+
+  &__code {
+    @include marginSetings;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondaryColor;
+  }
+
+  &__score-and-reviews {
+    @include marginSetings;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    height: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    color: $mainColor;
+  }
+
+  &__price {
+    @include marginSetings;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    height: 18px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondaryColor;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $mainColor;
+  }
+
+  &__buton-buy {
+    @include marginSetings;
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    background-color: $blueElements;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding